src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,76 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  position: relative;
+  background-color: var(--sbb-color-white-default);
+  width: 100%;
+  border-radius: var(--sbb-border-radius-4x);
+  padding-inline: var(--sbb-spacing-responsive-xxs-small);
+  overflow: hidden;
+  transition-property: box-shadow var(--sbb-animation-duration-4x) var(--sbb-animation-easing);
+
+  @include mq($from: medium) {
+    padding-inline: var(--sbb-spacing-responsive-xxs-medium);
+  }
+}
+
+:host([size='xs']) {
+  padding-block: var(--sbb-spacing-fixed-3x);
+}
+
+:host([size='s']) {
+  padding-block: var(--sbb-spacing-responsive-xxxs-small);
+
+  @include mq($from: medium) {
+    padding-block: var(--sbb-spacing-responsive-xxxs-medium);
+  }
+}
+
+:host([size='m']) {
+  padding-block: var(--sbb-spacing-fixed-3x);
```

Comment:
please don't redefine the properties in all variants, but only redefine a variable, see CODING_STANDARDS.md.